pennylane_lightning_kokkos/src/bindings/Bindings.cpp

```diff
@@ -65,21 +65,21 @@ void StateVectorKokkos_class_bindings(py::module &m) {
             return new StateVectorKokkos<PrecisionT>(num_qubits);
         }))
         .def(py::init([](std::size_t num_qubits,
-                         const Kokkos::InitArguments &kokkos_args) {
+                         const ::Kokkos::InitArguments &kokkos_args) {
             return new StateVectorKokkos<PrecisionT>(num_qubits, kokkos_args);
         }))
         .def(py::init([](const np_arr_c &arr) {
             py::buffer_info numpyArrayInfo = arr.request();
-            auto *data_ptr =
-                static_cast<Kokkos::complex<PrecisionT> *>(numpyArrayInfo.ptr);
+            auto *data_ptr = static_cast<::Kokkos::complex<PrecisionT> *>(
```

Comment:
Should we define somewhere
```c++
using CFP_t = ::Kokkos::complex<PrecisionT>;
```
and just reuse that?

Reply:
Agreed. That should probably be the best option, assuming it doesn't interfere elsewhere.

Reply:
Nice suggestion! Fixing this, Thanks @vincentmr!